src/System.Net.Libuv/src/project.json

```diff
@@ -1,12 +1,14 @@
-{
+﻿{
   "dependencies": {
-    "System.Diagnostics.Debug": "4.0.0", 
+    "System.Buffers": "0.1.0-d103015-1",
```

Comment:
These were the version numbers that were in the original nuspec files...we can change them easily now (just part of the VSO build parameters) if we so choose